wire-ios/Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Actions.swift

```diff
@@ -30,7 +30,6 @@ extension ZMConversationMessage {
     var canBeDigitallySigned: Bool {
         guard
             let selfUser = SelfUser.provider?.providedSelfUser,
-            selfUser.phoneNumber != nil,
```

Comment:
It would seem that the digital signature feature requires a phone number... as it will send an sms with a link. That suggests to me that either we can't delete the phone number, or instead backend needs to send the link via email.

Reply:
hmm, ok I'll check with backend... but as the phone number is no longer retrievable from the api, all new users will be without phone, no?

Reply:
So it seems it's unused and indeed broken, I am adding a todo here, so we clean it completely